cucoslib/workers/graphaggregator.py

```diff
@@ -21,8 +25,25 @@ def _handle_external_deps(self, ecosystem, deps):
         versions = solver.solve(deps)
         return [{"package": k, "version": v} for k, v in versions.items()]
 
+    def store_in_bucket(self, content):
+        bucket_name = 'janus-json-data'
+        try:
+            email = content.get('email')
+            s3 = AmazonS3(bucket_name=bucket_name)
+            s3.connect()
+
+            folder = 'new_user_profile'
+            key = folder + '/' + email + '.json'
+            s3.store_dict(content, key)
+        except:
+            raise Exception('Amazon Exception')
+
     def execute(self, arguments):
         self._strict_assert(arguments.get('manifest'))
+        self._strict_assert(arguments.get('user_profile'))
+
+        user_profile = arguments.get('user_profile')
```

Comment:
You can now directly access `user_profile` without `get()` method as the assert bellow guarantees existence.